easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'mpifileutils'
+version = "0.10.1"
+
+homepage = 'https://hpc.github.io/mpifileutils/'
+
+description = """
+ MPI-Based File Utilities For Distributed Systems
+"""
+
+toolchain = {'name': 'gompi', 'version': '2020a'}
+
+source_urls = ['https://github.com/hpc/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['mpifileutils-%(version)s_fix_acl_xattr_problem.patch']
+checksums = [
+    '4c8409ef4140f6f557d0e93f0c1267baf5d893c203b29fb7a33d9bc3c5a5d25c',  # v0.10.1.tar.gz
+    # mpifileutils-0.10.1_fix_acl_xattr_problem.patch
+    'e540ceaeb99758f4776252069d56b703411824ba83f48b474fc18785868151a2',
+]
+
+checksums = []
```

Comment:
@akesandgren This looks like a mistake? :)

Reply:
I wonder what new-pr did, there is a checksum in my file

Reply:
Ahhh, --inject-checksums has a bug!!!

Reply:
See https://github.com/easybuilders/easybuild-framework/issues/3459